### PR TITLE
Edits to Fix default setting and type for force_full_mesh. #6188

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 title: Optimizing subscriber-only groups
 navTitle: Optimizing
-description: 'Optimizing subscriber-only groups in EDB Postgres Distributed'
+description: 'Optimizing subscriber-only groups in EDB Postgres Distributed.'
 ---
 
 With PGD 5.6 and later, it's possible to optimize the topology of [subscriber-only groups](overview). 
 
-In this optimized topology, a small number of fully active nodes, the write leaders of the data groups, replicate changes to the group leaders of subscriber-only groups, these group leaders then replicate changes to the other members of its subscriber-only group.
+In this optimized topology, a small number of fully active nodes&mdash;the write leaders of the data groups&mdash;replicate changes to the group leaders of subscriber-only groups. These group leaders then replicate changes to the other members of its subscriber-only group.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx:9]: Is this what was meant? I didn't think the commas were enough to set off the phrase if so.

File: product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx [hunk 2]
```diff
@@ -32,42 +32,44 @@ The write leader nodes in other non-subscriber-only groups replicate changes to
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx:32]: In the para before this section head, I wanted to check whether the second sentence needed some additional punctuation or if there was some error, as it seemed to contradict itself.

File: product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx [hunk 3]
```diff
@@ -32,42 +32,44 @@ The write leader nodes in other non-subscriber-only groups replicate changes to
...
+To enable the optimized topology, you must have your data nodes in subgroups, with proxy routing enabled on the subgroups.
 You can then set the GUC [`bdr.force_full_mesh`](/pgd/5.6/reference/pgd-settings#bdrforce_full_mesh) to `off` to allow the optimization to be activated.
-Note: This GUC needs to be set in the postgresql.conf file on each data node and each node restarted for the change to take effect.
 
-If any requirements of the optimized topology aren't met, the nodes in a subscriber-only group automatically revert to the full mesh topology. When this happens, you'll find in the logs of the nodes in the cluster messages that explain why the optimization was not possible, such as:
+!!! Note
+    This GUC needs to be set in the `postgresql.conf` file on each data node and each node restarted for the change to take effect.
+
+If any requirements of the optimized topology aren't met, the nodes in a subscriber-only group revert to the full mesh topology. When this happens, you'll find in the logs of the nodes in the cluster messages why the optimization wasn't possible, such as:
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx:64]: Please check this edit. There looked to be extra words here. I think this is what was meant.

File: product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx:9]: sure.

File: product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
-If this is the case, the nodes in a subscriber only groups automatically revert to the full mesh topology.
+If any of these are the case, the nodes in subscriber-only groups revert to the full mesh topology.
 
-To get the benefit of the new SO group and node replication, you must have your data-nodes in subgroups, with proxy routing enabled on the sub groups.
+To get the benefit of the new SO group and node replication, you must have your data nodes in subgroups, with proxy routing enabled on the subgroups.
 
 ## How the optimized topology works
 
-For clusters using groups for their data nodes, subscriber only groups can use a more efficient model which uses a subscriber-only group leader, similar to a write leader in PGD Proxies.
+For clusters using groups for their data nodes, subscriber-only groups can use a more efficient model. This model uses a subscriber-only group leader, similar to a write leader in PGD proxies.
```
Comment (djw-m) [inline product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx:25]:
```suggestion
For clusters using groups for their data nodes, subscriber-only groups can use a more efficient model. This model uses subscriber-only group leaders, similar to write leaders in PGD proxies.
```

File: product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx:32]: Have made changes to make it clearer.

**Review by djw-m — APPROVED** [review]
LGTM - made other changes where pointed out.

**Pull request merged**